test/enforcements.spec.js

```diff
@@ -66,7 +66,6 @@ describe('Enforce some developments patterns', () => {
         [
           ['src/main/webapp', '<%= CLIENT_MAIN_SRC_DIR %>'],
           ['src/test/javascript', '<%= CLIENT_TEST_SRC_DIR %>'],
-          [' Java ', ' <%= backendName %> '],
```

Comment:
Why is it important to check, if the string 'Java" is inside the common folder files? I need to have java installed in the .devcontainer files. And I did not see value in this check so I removed it instead of excluding the .devcontainer-files from this test.

Reply:
We should not generate anything specific to server or client at common generator.
Ex: dotnetcore and nodejs blueprints probably uses the common generator.
Thinking about this, this devcontainer is java only and should be moved to the server generator.

Reply:
@mshima Can you please provide a suggestion for the best way to fix this?

Reply:
@mraible devcontainer files should be moved to the server generator. Since it’s specific to Java.